src/solver/mus.cpp

```diff
@@ -245,56 +256,56 @@ struct mus::imp {
         if (A.size() < B.size()) {
             expr_set::iterator it = A.begin(), end = A.end();
             for (; it != end; ++it) {
-                if (B.contains(*it)) return true;
+                if (B.contains(*it))
+                    return true;
             }
         }
         else {
             expr_set::iterator it = B.begin(), end = B.end();
             for (; it != end; ++it) {
-                if (A.contains(*it)) return true;
+                if (A.contains(*it))
+                    return true;
             }
         }
         return false;
     }
 
     bool is_core(expr_ref_vector const& mus) {
-        return l_false == m_solver.check_sat(mus); 
+        return l_false == m_solver.check_sat(mus);
     }
 
     class scoped_append {
         expr_ref_vector& m_fmls;
-        unsigned         m_size;
+        unsigned m_size;
+
     public:
-        scoped_append(imp& imp, expr_ref_vector& fmls1, expr_set const& fmls2):
-            m_fmls(fmls1),
-            m_size(fmls1.size()) {
+        scoped_append(imp& imp, expr_ref_vector& fmls1, expr_set const& fmls2)
+            : m_fmls(fmls1), m_size(fmls1.size()) {
             expr_set::iterator it = fmls2.begin(), end = fmls2.end();
             for (; it != end; ++it) {
                 fmls1.push_back(*it);
-            }            
+            }
         }
-        scoped_append(imp& imp, expr_ref_vector& fmls1, expr_ref_vector const& fmls2):
-            m_fmls(fmls1),
-            m_size(fmls1.size()) {
+        scoped_append(imp& imp, expr_ref_vector& fmls1,
+                      expr_ref_vector const& fmls2)
+            : m_fmls(fmls1), m_size(fmls1.size()) {
             fmls1.append(fmls2);
         }
-        scoped_append(imp& imp, expr_ref_vector& fmls1, ptr_vector<expr> const& fmls2):
-            m_fmls(fmls1),
-            m_size(fmls1.size()) {
+        scoped_append(imp& imp, expr_ref_vector& fmls1,
+                      ptr_vector<expr> const& fmls2)
+            : m_fmls(fmls1), m_size(fmls1.size()) {
             fmls1.append(fmls2.size(), fmls2.c_ptr());
         }
-        scoped_append(imp& imp, expr_ref_vector& fmls1, expr* fml):
-            m_fmls(fmls1),
-            m_size(fmls1.size()) {
+        scoped_append(imp& imp, expr_ref_vector& fmls1, expr* fml)
+            : m_fmls(fmls1), m_size(fmls1.size()) {
             fmls1.push_back(fml);
         }
         ~scoped_append() {
             m_fmls.shrink(m_size);
         }
     };
 
-    template<class T>
-    void display_vec(std::ostream& out, T const& v) const {
+    template <class T> void display_vec(std::ostream& out, T const& v) const {
```

Comment:
This seems arbitrary. Putting the template portion on a separate line shortens the next line and separates the template portion (polymorphism quantifier) from type signature.